src/javascript/app/pages/user/view_popup/view_popup.js

```diff
@@ -283,6 +283,8 @@ const ViewPopup = (() => {
             $container.find('#notice_ongoing').setVisibility(1);
         }
 
+        Clock.stopExternalTimer(is_ended); // stop/resume timer
```

Comment:
why not just set external timer to blank? e.g. `Clock. setExternalTimer();` should achieve the same thing